lib/Text/Markup/Rest.pm

```diff
@@ -13,7 +13,7 @@ our $VERSION = '0.24';
 
 # Find Python (process stolen from App::Info).
 my ($PYTHON, $RST2HTML);
-for my $exe (WIN32 ? 'python.exe' : 'python') {
+for my $exe (WIN32 ? 'python.exe' : 'python3') {
```

Comment:
Looks like maybe [it should be](https://docs.python.org/3/using/windows.html) `python3.exe` on Windows, too.

Reply:
Ack, changed in our branch.